src/components/Home.tsx

```diff
@@ -1,6 +1,11 @@
 import React, { FC } from "react";
 import { RouteComponentProps } from "@reach/router";
+import Homepage from "../components/Homepage/Container";
```

Comment:
you are importing Container but name is Homepage, is it ok?

Reply:
fixed